### PR TITLE
message to create an app if none found

### LINKED ISSUE #2020: When apps list is empty, print message with instructions on creating an app

### PR DESCRIPTION
**Pull request by MiguelMoll**
Fixes #2020

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[message to create an app if none found](https://trello.com/c/eeISSAfu/2183-message-to-create-an-app-if-none-found)

File: cmd/convox/apps.go [hunk 1]
```diff
@@ -87,6 +87,11 @@ func cmdApps(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
+	if len(apps) == 0 {
+		stdcli.Writef("No apps found, try creating one via `convox apps create`")
```
Comment (soulshake) [inline cmd/convox/apps.go]: Et la cohérence des majuscules, c'est pour les chiens?! :P

Comment (MiguelMoll) [inline cmd/convox/apps.go]: 🐶 🐕

Comment (soulshake) [inline cmd/convox/apps.go]: :hocho: :dog2:

**Pull request closed without merging**